inception-layer-docmetadata/src/main/java/de/tudarmstadt/ukp/inception/ui/core/docanno/sidebar/DocumentMetadataAnnotationSelectionPanel.java

```diff
@@ -254,7 +255,8 @@ protected void populateItem(ListItem<AnnotationListItem> aItem)
         
         List<AnnotationListItem> items = new ArrayList<>();
         for (AnnotationLayer layer : listMetadataLayers()) {
-            if (!DocumentMetadataLayerSupport.TYPE.equals(layer.getType())) {
+            if (!DocumentMetadataLayerSupport.TYPE.equals(layer.getType())
```

Comment:
If the layers are already filtered in `listMetadataLayers()`, why filter them again?

Reply:
I wondered about this, too.
I'll remove the whole if construct

Reply:
done